dist.ini

```diff
@@ -95,6 +95,8 @@ skip = ^MIME::Lite$
 perl                   = 5.008
 Regexp::Common         = 2013031301
 Config::Tiny           = 0
+Date::Format           = 0
+Date::Parse            = 0
 File::ShareDir         = 0
```

Comment:
never mind. I was thinking it'd be nice to avoid loading Date::__ entirely for processes like dmarc_httpd (which are long running and only validate), but I think it'd be too much bother and TimeDate is a venerable module that's unlikely to be leaky.